src/doc/gui_manual/WorkingWithFiles/Read_Write_Options.rst

```diff
@@ -110,6 +110,169 @@ all information related to the material configuration during the *open*. This wi
 lead to longer open times. A value of ``0`` here means to ignore any material
 information if present.
 
+PLOT3D
```

Comment:
You should use "VisIt_" instead of "VisIt". You should have a single space between sentences. I noticed 2 spaces between sentences primarily in the example vp3d file.

Reply:
The above comments apply in general to the documentation.